src/index.html

```diff
@@ -5,9 +5,43 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="body">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card_image"
+        src="./images/imac.jpeg"
+        alt="APPLE DEVICE"
+      >
+
+      <h3 class="card_title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
+      <span class="card_code">Product code: 195434</span>
+      <div class="card_rating">
+        <div class="stars stars--4">
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+          <div class="stars_star"></div>
+        </div>
+        <p class="card_reviews">Reviews: 5</p>
+      </div>
+
+      <div class="card_price">
+        <div class="card_price-title">Price:</div>
+        <div class="card_price-value">$2,199</div>
+      </div>
```

Comment:
use semantic tags for text content elements
```suggestion
      <div class="card_price">
        <p class="card_price-title">Price:</p>
        <p class="card_price-value">$2,199</p>
      </div>
```